src/components/Footer/Footer.module.css

```diff
@@ -190,7 +190,9 @@ form {
 .rounded-social-buttons .social-button.linkedin,
 .rounded-social-buttons .social-button.instagram,
 .rounded-social-buttons .social-button.github {
-  background: var(--white);
+  background: #2b2a2A;
```

Comment:
it would be better if you could use variables for these

Reply:
Hey, I have made the required variable changes